.github/workflows/deploy-tag.yml

```diff
@@ -0,0 +1,56 @@
+name: Deployment
+
+on:
+  push:
+    tags:
+      - "20.*"
+      - "21.*"
+      - "22.*"
+      - "23.*"
+      - "24.*"
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: npm install
+    - run: npm test
+    - name: Version and tag
+      run: |
+        export VER=$(echo "$GITHUB_REF" | sed -e 's/refs\/heads\///')
```

Comment:
This isn't working as expected

On my computer it does:

```sh
bash-3.2$ GITHUB_REF=refs/heads/feature-branch-1
bash-3.2$ echo "$GITHUB_REF" | sed -e 's/refs\/heads\///'
```

Reply:
Looks like `GITHUB_REF` is in the form `refs/tags/TAG` and not `refs/heads/TAG`.